ui/imports/shared/panels/CurveSeparatorWithText.qml

```diff
@@ -1,43 +1,45 @@
-import QtQuick 2.14
-import QtQuick.Layouts 1.14
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 
 import utils 1.0
 
 RowLayout {
-    id: root
-
-    property string text
+    property alias text: text.text
```

Comment:
Why alias? The text down below already has `text: root.text` You have to decide here :)

Reply:
fixed!